symengine/cwrapper.cpp

```diff
@@ -734,6 +780,12 @@ void ntheory_binomial(basic s, const basic a, unsigned long b)
     s->m = SymEngine::binomial(static_cast<const Integer &>(*(a->m)), b);
 }
 
+// Matrix Wrappers
+SymEngine::DenseMatrix* matrix_dense_matrix()
```

Comment:
That was actually something I wrote earlier, and had forgotten to remove. Now removed it.